fastchat/model/model_adapter.py

```diff
@@ -1256,6 +1256,19 @@ def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("starchat")
 
 
+class MistralAdapter(BaseModelAdapter):
+    """The model adapter for Mistral models"""
+
+    def match(self, model_path: str):
+        return "mistral" in model_path.lower()
+
+    def load_model(self, model_path: str, from_pretrained_kwargs: dict):
+        raise NotImplementedError()
```

Comment:
Delete this? Did you run the code successfully with this line?

Reply:
That’s an oversight as we deploy with vLLM. Fixed now